tests/spark/test_mleap_model_export.py

```diff
@@ -14,16 +16,32 @@
 from mlflow.tracking.artifact_utils import _download_artifact_from_uri
 from mlflow.tracking._model_registry import DEFAULT_AWAIT_MAX_SLEEP_SECONDS
 from tests.helper_functions import score_model_in_sagemaker_docker_container
+from tests.pyfunc.test_spark import get_spark_session
 
 
 from tests.spark.test_spark_model_export import (  # pylint: disable=unused-import
     model_path,
     iris_df,
-    spark_context,
     spark_model_iris,
     spark_custom_env,
 )
 
+_logger = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope="session", autouse=True)
+def spark_context():
+    conf = pyspark.SparkConf()
+    conf.set(
+        key="spark.jars.packages",
+        value=(
+            "ml.combust.mleap:mleap-spark-base_2.11:0.12.0,"
+            "ml.combust.mleap:mleap-spark_2.11:0.12.0"
+        ),
+    )
+    spark_session = get_spark_session(conf)
+    return spark_session.sparkContext
+
 
 @pytest.mark.large
 def test_model_deployment(spark_model_iris, model_path, spark_custom_env):
```

Comment:
`spark_model_iris` uses `spark_context` defined in this file, not the one in `tests/spark/test_spark_model_export.py`.